.clang-format

```diff
@@ -1,3 +1 @@
 BasedOnStyle: LLVM
```

Comment:
We can get rid of the entire file. The XcodeFormatter doesn't need a custom one.